core/app/models/spree/product.rb

```diff
@@ -307,6 +307,13 @@ def category
         first
     end
 
+    def display_price_in_currency(currency = nil)
```

Comment:
@aplegatt you can use `product.price_in(currency)` as described here: https://guides.spreecommerce.org/developer/core/products.html#multi-currency-support